android/app/src/main/res/layout/activity_speakers.xml

```diff
@@ -54,22 +64,23 @@
                     android:id="@+id/speaker_details_title"
                     android:layout_width="wrap_content"
                     android:layout_height="wrap_content"
-                    android:text="Title"
                     android:fontFamily="sans-serif-condensed"
-                    android:paddingBottom="@dimen/padding_small"
-                    android:textStyle="bold"
+                    android:text="Title"
                     android:textColor="@android:color/white"
+                    android:textStyle="bold"
+                    android:paddingBottom="@dimen/padding_small"
                     android:textSize="@dimen/heading_text_size" />
 
                 <TextView
                     android:id="@+id/speaker_details_designation"
                     android:layout_width="wrap_content"
                     android:layout_height="wrap_content"
-                    android:text="Subtitile"
                     android:ellipsize="end"
                     android:fontFamily="sans-serif-condensed"
                     android:paddingBottom="@dimen/padding_small"
                     android:maxLines="1"
+                    android:paddingBottom="2dp"
```

Comment:
@the-dagger This branch have merge conflicts, please see this inclusive of your requested changes. https://github.com/fossasia/open-event-android/pull/1144 . Thanks.